Libraries/Components/Touchable/Touchable.js

```diff
@@ -725,14 +771,14 @@ const TouchableMixin = {
     this.longPressDelayTimeout = null;
   },
 
-  _isHighlight: function(state) {
+  _isHighlight: function(state: State) {
     return (
       state === States.RESPONDER_ACTIVE_PRESS_IN ||
       state === States.RESPONDER_ACTIVE_LONG_PRESS_IN
     );
   },
 
-  _savePressInLocation: function(e) {
+  _savePressInLocation: function(e: LayoutEvent) {
```

Comment:
Later on in the function, we call `TouchEventUtils.extractSingleTouch`, which expects to be called with the native event of a `PressEvent`. So, I think this type should be `PressEvent` and not `LayoutEvent`.